### PR TITLE
Task dfl 2491 resource inspector improvements

### PR DESCRIPTION
**Pull request by p01**
Fixes for DFL-3676

### CODE REVIEW

File: src/resource-manager/resource_detail_view.js [hunk 1]
```diff
@@ -214,6 +226,9 @@ cls.ResourceDetailView.create_ui_widgets = function()
   });
 
   var text_search = window.views.resource_detail_view.text_search = new TextSearch();
+  var TEXT_RESOURCE_DETAIL_CONTAINER_CLASSNAMES = ["markup", "css", "script", "text"].map( function(v) {
```
Comment (hzr) [inline src/resource-manager/resource_detail_view.js]: Extra space before the function.

File: src/resource-manager/resource_service.js [hunk 2]
```diff
@@ -56,6 +48,19 @@ cls.ResourceInspector = function(network_logger)
...
   };
 
+  this.get_group_order = function()
+  {
+    this.get_views();
+    if (this.group_order == null && this.tree_view != null)
+    {
+      this.group_order = this.tree_view.get_group_order();
+      this.group_order_type_index = {};
+      this.group_order.forEach( function(g, i) {
```
Comment (hzr) [inline src/resource-manager/resource_service.js]: Extra space.

File: src/resource-manager/resource_service.js [hunk 3]
```diff
@@ -125,8 +139,10 @@ cls.ResourceInspector = function(network_logger)
           document_id_index[d.documentID] = i;
 
           // set depth, pivot_id and same_origin
-          var p = a[document_id_index[d.parentDocumentID]] || {pivot_id: d.windowID, depth: 0};
-          var id = p.pivot_id + "_" + d.documentID;
+          var p = a[document_id_index[d.parentDocumentID]];
```
Comment (hzr) [inline src/resource-manager/resource_service.js]: Lots of single letter variable names here, I have no idea what they mean.

File: src/resource-manager/resource_service.js [hunk 4]
```diff
@@ -270,11 +293,14 @@ cls.ResourceInspector = function(network_logger)
     {
       e.addClass(HIGHLIGHT_CLASSNAME);
 
+      if (window.Tooltips)
```
Comment (hzr) [inline src/resource-manager/resource_service.js]: I think you can rely on Tooltips here. Also, you can drop the `window.` for classes.

File: src/resource-manager/resource_style.css [hunk 5]
```diff
@@ -260,7 +262,15 @@ a.resource-detail-link:hover
 .resource-detail-font
 {
   width: 100%;
-  height: -o-content-size;
+  height: 100%;
+  border: 0;
+  font-size: 64px;
+  font-family: "the font";
```
Comment (hzr) [inline src/resource-manager/resource_style.css:269]: What's this?

Comment (hzr) [inline src/resource-manager/resource_style.css:269]: Ah, I get it now, a comment would be nice.

**Comment by chriskr** [comment]
Only small issue IMO is the case sensitive filter, other than that LGTM.

(missed the main review, didn't fully get the concept of pivot ids, know them in the context of quick sort, but here all items seem to have one?)

**Pull request merged**